src/videojs-contrib-hls.js

```diff
@@ -216,18 +216,31 @@ Hls.canPlaySource = function() {
  */
 Hls.supportsNativeHls = (function() {
   let video = document.createElement('video');
-  let xMpegUrl;
-  let vndMpeg;
 
   // native HLS is definitely not supported if HTML5 video isn't
   if (!videojs.getComponent('Html5').isSupported()) {
     return false;
   }
 
-  xMpegUrl = video.canPlayType('application/x-mpegURL');
-  vndMpeg = video.canPlayType('application/vnd.apple.mpegURL');
-  return (/probably|maybe/).test(xMpegUrl) ||
-    (/probably|maybe/).test(vndMpeg);
+  // HLS manifests can go by many mime-types
+  let canPlay = [
```

Comment:
why not just have this be a list of strings and then in the return statement do a `map` before the `some`?